easybuild/easyconfigs/g/GffCompare/GffCompare-0.11.6-GCCcore-8.3.0.eb

```
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.11.6'
+
+homepage = 'https://github.com/gpertea/gffcompare'
+description = """GffCompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://ccb.jhu.edu/software/stringtie/dl/']
```

```suggestion
source_urls = ['https://ccb.jhu.edu/software/stringtie/dl/']
```
despite them redirecting you to http if you go browse the directory, if you specify full path to the source, it seems to actually stick to https, so, i think this works.